types/Area.d.ts

```diff
@@ -0,0 +1,104 @@
+import { Broadcastable } from './Broadcast';
+import { GameEntity } from './GameEntity';
+import { GameState } from './GameState';
+import { Npc } from './Npc';
+import { Room } from './Room';
+
+export declare class Area extends GameEntity {
+    /** Bundle this area comes from */
+    bundle: string;
+    /** @property {string} name */
+    name: string
+    /** @property {string} title */
+    title: string;
+    /** @property {string} script A custom script for this item */
+    script: string;
+    /** @property {Map<number, number>} map a Map object keyed by the floor z-index, each floor is an array with [x][y] indexes for coordinates. */
+    map: Map<number, number>;
```

Comment:
This is actually a map of `<number, AreaFloor>` where the number is the z-index.

Reply:
should be fixed now